wagtailio/navigation/templatetags/navigation_tags.py

```diff
@@ -4,7 +4,7 @@
 
 
 @register.inclusion_tag("patterns/includes/menu_footer.html", takes_context=True)
-def footer_navigation(context):
+def footer_navigation(context, block):
```

Comment:
This change here allows us to map a value over in the pattern library. We could also achieve this with context overrides, up to you what we do really.